clippy_lints/src/needless_braces_on_range_literal.rs

```diff
@@ -0,0 +1,67 @@
+//! Checks for braces on literals in range statements
+//!
+//! For example, the lint would catch
+//!
+//! ```rust
+//! for i in (0)..10 {
+//!   println!("{i}");
+//! }
+//! ```
+//!
+//! And suggest this:
```

Comment:
```suggestion
//! Use instead:
```
For consistency with other lints :)